### PR TITLE
Update cloud-recording.md

### PR DESCRIPTION
**Pull request by SamChung-MS**
Fixed the text around restricted users not being able to record. That's not true. They can record. They just won't have their recordings stored in Stream.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/cloud-recording.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ Learn more about how you can [assign licenses to users in Microsoft 365 or Offic
 
 ### Make sure users have upload video permissions in Microsoft Stream
 
-By default, everyone in the company can create content in Stream, once Stream is enabled and the license is assigned to the user. A Microsoft Stream administrator can [restrict employees for creating content](https://docs.microsoft.com/stream/restrict-uploaders) in Stream. The users who are in this restricted list will not be able to record meetings.
+By default, everyone in the company can create content in Stream, once Stream is enabled and the license is assigned to the user. A Microsoft Stream administrator can [restrict employees for creating content](https://docs.microsoft.com/stream/restrict-uploaders) in Stream. The users who are in this restricted list will still be able to record meetings, however, their recordings will not be sent to Stream. These recordings will instead be temporarily stored on AMS for 21 days. During which time, the recording must be downloaded before it is deleted at the end of those 21 days.
```
Comment (JohanFreelancer9) [inline Teams/cloud-recording.md:67]:
```suggestion
By default, everyone in the company can create content in Stream, once Stream is enabled and the license is assigned to the user. A Microsoft Stream administrator can [restrict employees for creating content](https://docs.microsoft.com/stream/restrict-uploaders) in Stream. The users who are in this restricted list will still be able to record meetings, however, their recordings will not be sent to Stream. These recordings will instead be temporarily stored on AMS for 21 days, during which time the recording must be downloaded before it is deleted at the end of those 21 days.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by bwilsonms** [comment]
Please note, I think this edit changed the syntax enough to cause a problem under this section
image

**Comment by SamChung-MS** [comment]
@bwilsonms I submitted a new PR with a fix. Just as you suspected, looks like the powershell tag wasn't closed properly.

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by bwilsonms — APPROVED** [review]
Looks good.

**Pull request merged**